client/app/constants/SpecialIssues.js

```
@@ -371,7 +373,7 @@ export const NEW_SPECIAL_ISSUES = [
     display: 'US Court of Appeals for Veterans Claims (CAVC)',
     queueDisplay: 'US Court of Appeals for Veterans Claims (CAVC)',
     specialIssue: 'usCourtOfAppealsForVeteransClaims',
-    snakeCase: 'cavc',
+    snakeCase: 'us_court_of_appeals_for_veterans_claims',
```

match the `specialIssue` name

---

Why?

---

When we pass from the Javascript code back to the Ruby code, we map these to each other through a snakecase function. if they don't match, we silently fail and drop the param.

`specialIssue` is the javascript variable name, `snakeCase` is the ruby variable name. See the topline comment for the full breakdown of these fields' uses!

---

If they don't match, shouldn't we log an error?